packages/editor/src/client/atoms/dateTimePicker.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import {DatePicker, FormGroup, ControlLabel, Form} from 'rsuite'
+import {useTranslation} from 'react-i18next'
+
+export interface DateTimeProps {
```

Comment:
I would rename it to `DateTimePickerProps`. It's the properties for the `DateTimePicker` component